int32 x_min
int32 x_max
int32 y_min
int32 y_max
---
geometry_msgs/PointStamped point_stamped
